source/index.rst

```diff
@@ -7,31 +7,20 @@ Adventure
    .. warning:: This documentation is for an **unreleased** version of 
       Adventure. Some information may not be up to date, and API is subject to change.
 
-.. toctree::
-   :maxdepth: 2
-
-   audiences
-   text
-   serializer/index
-
-   bossbar
-   sound
-   title
-   book
-   minimessage
 
-   platform/index
+Adventure is a library for server-controllable user interface elements in *Minecraft: Java Edition*. The different
+elements covered by this library can be found in the index on the left, or on the bottom of this page.
 
-   migration/index
+Check out the :doc:`quickstart` section for a little introduction to the most basic functionality in adventure
 
-
-Adventure is a library for server-controllable user interface elements in *Minecraft: Java Edition*.
+If you are here to troubleshoot an error commonly addressed errors can be found with answers over at the :doc:`troubleshooting`
+page.
 
 
 Importing Adventure into your project
 -------------------------------------
 
-First, add the repository:
+First, add the repository(if necessary):
```

Comment:
Could be worth mentioning here that Adventure can be found in Maven Central. Tbh it might be worth just moving snapshot builds into its own section as the large majority of devs won't need this.